src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocstyle/InputJavadocStyle8.java

```diff
@@ -0,0 +1,389 @@
+    */
+   void method7() {} // ok
+
+   /**
+    * Public check should fail</code>
+    * should fail <
+    */
+   public void method8() {} // ok
+
+   /** {@inheritDoc} **/
+   public void method9() {} // ok
+
+    // Testcases to exercise the Tag parser (bug 843887)
+
+    /**
+     * Real men don't use XHTML.
+     * <br />
+     * <hr/>
+     * < br/>
+     * <img src="schattenparker.jpg"/></img>
+     */
+    private void method10() {} // ok
+
+    /**
+     * Tag content can be really mean.
+     * <p>
+     * Sometimes a p tag is closed.
+     * </p>
+     * <p>
+     * Sometimes it's not.
+     *
+     * <span style="font-family:'Times New Roman',Times,serif;font-size:200%">
+     * Attributes can contain spaces and nested quotes.
+     * </span>
+     * <img src="slashesCanOccurWithin/attributes.jpg"/>
+     * <img src="slashesCanOccurWithin/attributes.jpg">
+     * <!-- comments <div> should not be checked. -->
+     */
+    private void method11() {} // ok
+
+    /**
+     * Tags for two lines.
+     * <a href="some_link"
+     * >Link Text</a>
+     */
+    private void method12() {} // ok
+
+    /**
+     * First sentence.
+     * <pre>
+     * +--LITERAL_DO (do)
+     *     |
+     *     +--SLIST ({)
+     *         |
+     *         +--EXPR
+     *             |
+     *             +--ASSIGN (=)
+     *                 |
+     *                 +--IDENT (x)
+     *                 +--METHOD_CALL (()
+     *                     |
+     *                     +--DOT (.)
+     *                         |
+     *                         +--IDENT (rand)
+     *                         +--IDENT (nextInt)
+     *                     +--ELIST
+     *                         |
+     *                         +--EXPR
+     *                             |
+     *                             +--NUM_INT (10)
+     *                     +--RPAREN ())
+     *         +--SEMI (;)
+     *         +--RCURLY (})
+     *     +--LPAREN (()
+     *     +--EXPR
+     *         |
+     *         +--LT (<)
+     *             |
+     *             +--IDENT (x)
+     *             +--NUM_INT (5)
+     *     +--RPAREN ())
+     *     +--SEMI (;)
+     * </pre>
+     */
+    private void method13() {} // ok
+
+    /**
+     * Some problematic javadoc. Sample usage:
+     * <blockquote>
+     */
+
+    private void method14() {} // ok
+
+    /**
+     * Empty line between javadoc and method declaration cause wrong
+     * line number for reporting error (bug 841942)
+     */
+
+    private void method15() {} // ok
+
+    /** Description of field: {@value}. */
+    public static final int dummy = 4911; // ok
+
+    /**
+     */
+    public void method16() {} // violation
+
+    /**
+     * @param a A parameter
+     */
+    protected void method17(String a) {} // violation
+
+    /**
+     * @exception RuntimeException should be thrown
+     */
+    void method18(String a) {} // violation
+
+    /**
+     */
+    private static int ASDF = 0; // ok
```

Comment:
says ok, but expected violation here in test case

Reply:
yeah I wrote this by mistake.

Reply:
done